content/docs/capabilities/ppl.mdx

```diff
@@ -34,19 +34,19 @@ Each PPL policy has at the top level a set of `allow` or `deny` actions, with a
 ```yaml
 allow:
   and:
-    - domain:
-        is: example.com
-    - user:
+    - email:
```

Comment:
If we're going to have just one condition in this block I'd recommend keeping the `domain` condition rather than `email`. I think that makes a little more sense as a policy: "allow everyone from this domain except for these two specific users"; otherwise it's not clear why you would need a `deny` block at all.